src/wdio/services/SeleniumDockerService.js

```diff
@@ -65,39 +31,125 @@ export default class SeleniumDockerService {
     this.port = config.port;
     this.path = config.path;
     this.browserName = capabilities[0].browserName;
-    this.cidfile = this.config.cidfile || this.cidfile;
+    this.container = await this.getRunningContainer();
```

Comment:
Do we even need to do this logic if we're disabled?

Reply:
no, we can exit early